scipy/optimize/_shgo.py

```diff
@@ -77,9 +77,14 @@ def shgo(func, bounds, args=(), constraints=None, n=100, iters=1, callback=None,
     n : int, optional
         Number of sampling points used in the construction of the simplicial
         complex. Note that this argument is only used for ``sobol`` and other
-        arbitrary `sampling_methods`.
+        arbitrary `sampling_methods`. In case of ``sobol``, it must be a
+        power of 2: ``n=2**m``, and the argument will automatically be
+        converted to the next higher power of 2. Default is 100 for
+        ``sampling_method='simplicial'`` and 128 for
+        ``sampling_method='sobol'``.
     iters : int, optional
-        Number of iterations used in the construction of the simplicial complex.
+        Number of iterations used in the construction of the simplicial
+        complex. Default is 1.
```

Comment:
The default is already specified in the function signature, not really needed here.

Reply:
But I thought we must add the default in the description as well?? We do this almost everywhere and I remember reviewers asking for this in the past on another PR.